spec/unit/models/services/service_instance_operation_spec.rb

```diff
@@ -51,5 +51,37 @@ module VCAP::CloudController
         expect(operation.state).to eq 'finished'
       end
     end
+
+    describe 'when two are created with the same id' do
+      describe 'when a ServiceInstanceOperation exists' do
+        let(:service_instance_attrs) do
+          {
+            name: 'bommel_instance',
+            space: VCAP::CloudController::Space.make
+          }
+        end
+
+        let(:service_instance) { ServiceInstance.create(service_instance_attrs) }
+        let(:operation_attributes2) do
+          {
+            service_instance_id: service_instance.id,
+            state: 'in progress',
+            description: '50% all the time',
+            type: 'create',
+            proposed_changes: {
+              name: 'pizza',
+              service_plan_guid: '1800-pizza',
+            },
+          }
+        end
+        before { ServiceInstanceOperation.create(operation_attributes2) }
```

Comment:
You can use `ServiceInstanceOperation.make(service_instance_id: service_instance.id)` and remove `operation_attributes2`.